source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer/AdaptiveFileBrowserSource.mm

```diff
@@ -45,7 +46,13 @@ - (instancetype)initWithFrame:(CGRect)frame
         if (hostconfigParseResult.isValid) {
             _hostConfig = hostconfigParseResult.config;
         }
-
+#if RELEASE
+        // the most of cards below v1.3 don't have updated accssibility features, but these cards do serve well as visualization test during bug bash
+        // so instead of removing them, excluded these cards when built for release.
+        _restrictedPaths = [NSSet setWithObjects:@"v1.0", @"v1.1", @"HostConfig", @"Templates", @"Elements", @"Tests", nil];
```

Comment:
did you miss v1.2?

Reply:
no, I found there were no updated cards for some of the scenario cards in v1.2, and fortunately, the scenario cards of v1.2 were updated with accessibility features.

Reply:
can you reword `the most of cards below v1.3 don't have updated accssibility features`

Reply:
sure & done!